xpdacq/xpdacq_conf.py

```diff
@@ -180,14 +185,47 @@ def configure_frame_acq_time(new_frame_acq_time):
     print("INFO: area detector has been configured to new "
           "exposure_time = {}s".format(new_frame_acq_time))
 
-def _load_beamline_config(beamline_config_fp):
+def _verify_within_test(beamline_config_fp,verif):
+    while verif != "y":
+        with open(beamline_config_fp, 'r') as f:
+            beamline_config = yaml.load(f)
+        warnings.warn("Not verified")
+        verif = "y"
+    beamline_config["Verified by"] = "AUTO VERIFIED IN TEST"
+    timestamp = datetime.datetime.now()
+    beamline_config["Verification time"] = timestamp.strftime('%Y-%m-%d %H:%M:%S')
+    with open(beamline_config_fp, 'w') as f:
+        yaml.dump(beamline_config,f)
+    return beamline_config
+    
+
+def _load_beamline_config(beamline_config_fp, verif = "", test=False):
     if not os.path.isfile(beamline_config_fp):
         raise xpdAcqException("WARNING: can not find long term beamline "
                               "configuration file. Please contact the "
                               "beamline scientist ASAP")
-    with open(beamline_config_fp, 'r') as f:
-        beamline_config = yaml.load(f)
-
+    pp = pprint.PrettyPrinter()
+    os_type = platform.system()
+    if os_type == 'Windows':
+        editor = 'notepad'
+    else:
+        editor = os.getenv('EDITOR')
+    if not test:
+        while verif != "y":
```

Comment:
It might not be a bad idea to accept ``Y`` in addition to ``y``(or even ``yes`` and ``Yes`` if you want to be flexible) since sometimes user will really type what they want, but not what we listed.

Reply:
Sure.

Reply:
you might want to cast everything to lower/upper before checking it.